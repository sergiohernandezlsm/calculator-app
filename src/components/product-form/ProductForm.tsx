import React from 'react';
import Form from 'react-bootstrap/Form';
import FornInput from '../form-input';
import styles from './ProductForm.module.scss';

export interface Props {
  calculatorHandler: (x: any) => void;
}

const ProductForm: React.FC<Props> = (props) => {
  return (
    <div className={styles.inputWrapper}>
      <Form>
        <FornInput calculatorHandler={props.calculatorHandler} placeholder="Duration" text={'(in %)'} name={'Interes rate'} />
      </Form>
    </div>
  )
}

export default ProductForm;