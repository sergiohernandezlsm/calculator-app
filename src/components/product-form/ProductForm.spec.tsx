import { shallow } from 'enzyme';
import ProductForm from './ProductForm';

describe('ProductForm Component', () => {
  it('renders <ProductForm /> component', () => {
    const wrapper = shallow(<ProductForm />);
    expect(wrapper).toMatchSnapshot();
  });
});