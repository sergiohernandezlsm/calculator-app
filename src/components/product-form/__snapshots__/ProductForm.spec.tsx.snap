// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductForm Component renders <ProductForm /> component 1`] = `ShallowWrapper {}`;
